src/main/resources/templates/db_scripts/data-migration/po_line_table.sql

```diff
@@ -0,0 +1,12 @@
+UPDATE ${myuniversity}_${mymodule}.po_line
+SET jsonb =
+    (
+      -- Update each fundDistribution element renaming 'percentage' to 'value' and adding 'distributionType' with 'percentage' value
+      SELECT jsonb_set(jsonb, '{fundDistribution}', jsonb_agg(distrib - 'percentage' || jsonb_build_object('value', coalesce(distrib -> 'percentage', distrib -> 'value'), 'distributionType', coalesce(distrib ->> 'distributionType', 'percentage'))))
```

Comment:
Possible typo:  `distrib - 'percentage'`  Should this be an arrow?

Reply:
Actually it is one of [jsonb operators](https://www.postgresql.org/docs/10/functions-json.html#FUNCTIONS-JSONB-OP-TABLE) to delete key/value pair

Reply:
got it.  Wasn't familiar with that operator.  Thanks for clarifying.